rules/windows/credential_access_adidns_wildcard.toml

```diff
@@ -0,0 +1,82 @@
+[metadata]
+creation_date = "2024/03/26"
+integration = ["system", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Active Directory Integrated DNS (ADIDNS) is one of the core components of AD DS, leveraging AD's access control and
+replication to maintain domain consistency. It stores DNS zones as AD objects, a feature that, while robust, introduces
+some security issues, such as wildcard records, mainly because of the default permission (Any authenticated users) to
+create DNS-named records. Attackers can create wildcard records to redirect traffic that doesn't explicitly match
+records contained in the zone, becoming the Man-in-the-Middle and being able to abuse DNS similarly to LLMNR/NBNS spoofing.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential ADIDNS Poisoning via Wildcard Record Creation"
```

Comment:
```suggestion
name = "Wildcard Record Creation via Potential ADIDNS Poisoning"
```

Reply:
Hmmm I don't think this one makes sense here, the wildcard record is intended to enable the ADIDNS attack